api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
Was this also done by the go dependency updates? Or is adding this change accidental?

Reply:
I am not sure. When I build after updating dependencies it ended up with this diff.

Reply:
It seems that both tags need to be present (the first one for gofmt >= 1.17 and the second one for compatibility), so I think the change is anyway good: https://github.com/kubernetes/gengo/pull/210